packages/front-end/pages/saved-groups.tsx

```diff
@@ -88,13 +89,7 @@ export default function SavedGroupsPage() {
           // Ignore errors
         });
     }
-  }, [
-    permissions.manageTargetingAttributes,
-    apiCall,
-    refreshOrganization,
-    attributeSchema,
-    savedGroups,
-  ]);
+  }, [apiCall, refreshOrganization, attributeSchema, savedGroups, permissions]);
```

Comment:
`permissions` is not memoized and will change every time the app renders.  We should update `UserContext.tsx` and memoize it there since this is likely not the only place doing this.